lib/components/viewers/related-stops-panel.js

```diff
@@ -111,6 +113,12 @@ class RelatedStopsPanel extends Component {
                           key={id}
                           pattern={pattern}
                           route={route}
+                          routeColor={getRouteColorBasedOnSettings(
```

Comment:
Is there a reason why next arrivals in this panel are separate, whereas those for that stop are "glued" together?

Reply:
Aesthetics. If you glue the small ones together things look very busy if there are multiple colors going on and very messy if all lines are the same color. With the larger stop blocks we can rely on the visual hierarchy to help out more but in the small list we're only working with one horizontal row so things get muddy fast.

Reply:
I still think "gluing" the arrivals by stop would provide visual separation between nearby stops. The headings don't outline groups of arrivals as nicely.

Reply:
I think the real solution is to move to the nearby view we've been discussing for some time. The entire idea of the "nearby stops" view below the stop has never sat right with me.